### PR TITLE
braindamage

### PR DESCRIPTION
**Pull request by Amelia0010**
rebwase when

#### Changelog

:cl:  
tweak: changed some brain damage lines
/:cl:

### CODE REVIEW

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -51,7 +51,10 @@
 		"TURBIN IS BEST ENGIENE",
 		"SOLIRS CAN POWER THE HOLE STATION ANEWAY",
 		"parasteng was best",
-		"Tajaran has warrrres, if you have coin"
+		"Tajaran has warrrres, if you have coin",
+		"MAEK IT A PWEFEREENCE!!!",
```
Comment (Super3222) [inline strings/brain_damage_lines.json:55]: :+1:

File: strings/brain_damage_lines.json [hunk 2]
```diff
@@ -140,7 +150,7 @@
 		"SILICONS",
 		"BLOB"
 	]
-	
+
 	"salt": [
 		"ReBase TO @pick(CODEBASE)",
 		"REVIRT TO OLD @pick(CODEBASE)",
```
Comment (AshCorr) [inline strings/brain_damage_lines.json:157]: REVIRT TO OLD OLD YOGS

File: strings/brain_damage_lines.json [hunk 3]
```diff
@@ -51,7 +51,10 @@
 		"TURBIN IS BEST ENGIENE",
 		"SOLIRS CAN POWER THE HOLE STATION ANEWAY",
 		"parasteng was best",
-		"Tajaran has warrrres, if you have coin"
+		"Tajaran has warrrres, if you have coin",
+		"MAEK IT A PWEFEREENCE!!!",
+		"MAKE FART COMM SCRIPT XDXDXDDXXDDD",
+		"X-X-X-ENO E-V-V-ENT P-P-P-LS!!"
```
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [inline strings/brain_damage_lines.json]: X-X-X-X-X-X-X-X-X-X-X-X-E-E-E-E-E-N-N-N-N-O-O-O-O-O E-E-E-E-E-E-E-E-E-E-E-E-E-E-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-V-E-E-E-N-N-T-T-T-T P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-P-L-L-L-L-S-S!!

File: strings/brain_damage_lines.json (hunk 2 above)
Comment (Amelia0010) [reply strings/brain_damage_lines.json:157]: intended feature. It is braindamage, after all

**Comment by Amelia0010** [comment]
https://images-ext-2.discordapp.net/external/WA6T9_0Mo4li7TUmlNuI7bq1FuUJhF-Qq_LY5AoIRcw/https/media.discordapp.net/attachments/134722036353204224/368971771115864065/DMScneK.png

**Pull request merged**